### PR TITLE
Update Xcode source completion sample

### PR DESCRIPTION
**Pull request by jaroshevskii**
Swift prefers camelCase style for functions.

### Why:

This change ensures consistency with Swift's preferred camelCase naming convention for function names, aligning the documentation with the official Swift coding guidelines.

### What's being changed:

The example function signature in the documentation is updated from PascalCase (CalculateDaysBetweenDates) to camelCase (calculateDaysBetweenDates). This adjustment reflects the naming conventions used in Swift:

**Before:**

```swift
func CalculateDaysBetweenDates(
```

**After:**

```swift
func calculateDaysBetweenDates(
```

### Check off the following:

- [x] A subject matter expert (SME) has reviewed the technical accuracy of the content in this PR. In most cases, the author can be the SME. Open source contributions may require a SME review from GitHub staff.
- [x] The changes in this PR meet [the docs fundamentals that are required for all content](http://docs.github.com/en/contributing/writing-for-github-docs/about-githubs-documentation-fundamentals).
- [x] All CI checks are passing.

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@jaroshevskii Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md [hunk 1]
```diff
@@ -445,8 +445,8 @@ This guide demonstrates how to get coding suggestions from {% data variables.pro
 {% data variables.product.prodname_copilot %} offers coding suggestions as you type. For example, type this function
 signature in a Swift file:
 
-```shell copy
-func CalculateDaysBetweenDates(
+```text copy
```
Comment (felicitymay) [inline content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md:448]: I think that `shell` may have been a typo. It probably makes sense to make the following update, to align with examples for other IDEs.

```suggestion
```swift copy
```

**Review by felicitymay — DISMISSED** [review]
Many thanks for raising this pull request to improve our docs. I particularly appreciated your clear description of the problem that you were fixing. ✨ 

I've made one suggestion to change the language for the code block to Swift, but otherwise this looks good to merge. 🚀

File: content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md [hunk 2]
```diff
@@ -445,8 +445,8 @@ This guide demonstrates how to get coding suggestions from {% data variables.pro
 {% data variables.product.prodname_copilot %} offers coding suggestions as you type. For example, type this function
 signature in a Swift file:
 
-```shell copy
-func CalculateDaysBetweenDates(
+```swift copy
```
Comment (felicitymay) [inline content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md:448]: Now I see why you didn't use Swift. I'll check why that's not included in our supported languages.

**Review by Moo021781 — COMMENTED** [review]
I don't know

File: content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md (hunk 2 above)
Comment (felicitymay) [reply content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md:448]: I've opened:

- https://github.com/github/docs/pull/35732

Once that's approved and merged, we can update your branch and merge this.

**Review by felicitymay — APPROVED** [review]

File: content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md (hunk 2 above)
Comment (felicitymay) [reply content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md:448]: That looks better 🙂 

Screenshot showing the updated content in the preview.

**Comment by felicitymay** [comment]
Thanks for your patience @jaroshevskii ✨ 

My pull request needed a reviewer who's in a later time zone than me. However, that's now merged and your PR is also in the merge queue. Your update will take a little while to get to production, but should be there before the weekend.

**Pull request merged**